test/libsolidity/semanticTests/externalContracts/deposit_contract.sol

```diff
@@ -196,13 +196,15 @@ contract DepositContract is IDepositContract, ERC165 {
 // gas legacy: 128065
 // gas legacyOptimized: 100398
 // get_deposit_count() -> 0x20, 8, 0
-// deposit(bytes,bytes,bytes,bytes32), 1 ether: 0x80, 0xe0, 0x120, 0xaa4a8d0b7d9077248630f1a4701ae9764e42271d7f22b7838778411857fd349e, 0x30, 0x933ad9491b62059dd065b560d256d8957a8c402cc6e8d8ee7290ae11e8f73292, 0x67a8811c397529dac52ae1342ba58c9500000000000000000000000000000000, 0x20, 0x00f50428677c60f997aadeab24aabf7fceaef491c96a52b463ae91f95611cf71, 0x60, 0xa29d01cc8c6296a8150e515b5995390ef841dc18948aa3e79be6d7c1851b4cbb, 0x5d6ff49fa70b9c782399506a22a85193151b9b691245cebafd2063012443c132, 0x4b6c36debaedefb7b2d71b0503ffdc00150aaffd42e63358238ec888901738b8 -> # txhash: 0x7085c586686d666e8bb6e9477a0f0b09565b2060a11f1c4209d3a52295033832 #
```

Comment:
Why did you remove the txhash comment?

Reply:
Looks like that this was removed accidentally. Need to check why exactly.

Reply:
The current implementation of the side effects seem to have problems with comments.

Reply:
Ok, fixed. Comments work now with side effects.